### ISSUE TITLE
Relative path is not resolved correctly

### ISSUE THREAD

**Issue by fenuks** [IBM/openapi-validator#199]
With version 0.30.0, compared to 0.29.4, openapi-validator regressed in way how relative paths are resolved.

I have following folder structure:
```sh
src/
    api1.yml
    api2.yml
    shared/
        common.yml
```
`api1.yml` and `api2.yml` files reference some definitions from `shared/common.yml`, e.g. responses `shared/common.yml#/components/responses/NoContent`.

If I run validator from other folder than `src/` then e.g. `lint-openapi src/api1.yml`, then error is reported.

```sh
errors
  Message :   ENOENT: no such file or directory, open 'shared/common.yml'
  Path    :   paths./mymethod.post.responses.201.$ref
  Line    :   42
```
It seems as if relative path is now resolved in respect to current working directory, instead of relatively to specification file being processed.

Earlier versions work fine.

**Comment by dpopp07** [comment]
@fenuks I'm surprised to see this, thank you for reporting the bug.

@jorge-ibm this may have to do with the Spectral integration - do you mind taking a look at it?

**Comment by fenuks** [comment]
No problem. I'm using this program extensively, thank you for all hard work!

And I forgot to mention that running linter from the same directory where processed file is, naturally works as expected.

**Comment by jorge-ibm** [comment]
This issue should be fixed as part of this PR https://github.com/IBM/openapi-validator/pull/201
